src/ga4gh/vrs/extras/vcf_annotation.py

```diff
@@ -90,7 +88,7 @@ class SeqRepoProxyType(str, Enum):
     help="The assembly that the `vcf_in` data uses.",
     type=str
 )
-def annotate_click(
+def annotate_click(  # pylint: disable=too-many-arguments
```

Comment:
I think if it's fewer than this many, it's okay
https://github.com/chrchang/plink-ng/blob/master/1.9/plink.c#L322

Reply:
😂